step1_masking_experiment.py

```diff
@@ -126,9 +125,9 @@
 #I need to find out whether we need 2 different trial types (left vs right, or category A or B)
 #or 4 different trial types 2 (direction) X 2 (category)
 
-if task_local == True:
+if task_local:
 	complexity += 1
-if task_disc == True:
+if task_disc:
 	complexity += 1
 
```

Comment:
Is this the same as:
```suggestion
if task_local or task_disc:
	complexity += 1
```
Unless a task can be both a localization and discrimination? But I don't think so

Reply:
Nevermind haha. Ignore this